cohortextractor/tpp_backend.py

```diff
@@ -1830,6 +1830,25 @@ def patients_with_test_result_in_sgss(
             column = f"CASE {case_clauses} ELSE {raw_column} END"
         elif returning == "variant_detection_method":
             column = "t2.variant_detection_method"
+        elif returning == "symptomatic":
+            raw_column = "t2.symptomatic"
+            transforms = {
+                # From SGSS_AllTests_Positive table.
+                "N": "N",
+                "U": "",
+                "Y": "Y",
+                # From SGSS_AllTests_Negative table.
+                "NULL": "",
```

Comment:
Are you matching the string `"NULL"` or a literal SQL `NULL`?

Reply:
Think I've fixed this now.